python/ray/util/annotations.py

```diff
@@ -107,22 +123,50 @@ def Deprecated(*args, **kwargs):
     if len(args) == 1 and len(kwargs) == 0 and callable(args[0]):
         return Deprecated()(args[0])
 
-    message = (
-        "**DEPRECATED:** This API is deprecated and may be removed in a future "
-        "Ray release."
+    doc_message = (
+        "\n    DEPRECATED: This API is deprecated and may be removed "
+        "in future Ray releases."
+    )
+    warning_message = (
+        "This API is deprecated and may be removed in future Ray releases. "
+        "You could suppress this warning by setting env variable "
+        'PYTHONWARNINGS="ignore::DeprecationWarning"'
```

Comment:
Should this be RayDeprecationWarning?

Reply:
no; since DeprecationWarning extends DeprecationWarning